### ISSUE TITLE
OopsAllLalafellsSRE does not do the same thing as Potato Famine 2

### ISSUE THREAD

**Issue by frukbachers** [Aida-Enna/XIVPlugins#32]
https://github.com/kelvin124124/OopsAllLalafellsSRE

OopsAllLalafellsSRE is not doing the same thing as Potato Famine 2.

It changes anyone to the target race and I want just Lalafells to be some other race. It's exactly as you stated in your mission statement when it was recent: people do nasty stuff with lalafells and even when not in a syncshell via Mare, you can see them doing nasty stuff.

Obviously I cannot force you to do so but please reconsider, the feature offerd by the other plugin is undeniably different from what was offered by you.

**Comment by Sharinunu** [comment]
I agree with OP. 

I tried messing around with it but the only way to change them is to change every single player to 1 race, which is not really the same thing as before.

I do hope that you reconsider please :(  I know perhaps it's not the most used plug-in and may not be worth the hassle for a few people but it's a game changer and really makes the experience a lot healthier for those of us who do use it!

I tried my best not to sound too demanding and I'm sorry if I failed at that :(

**Comment by Korunos47** [comment]
I agree with this statement too. It is unfortunately not the same.
I hope you reconsider your choice...
It would be nice to have this plugin back!

I am losing my mind with the emote spamming lala's

**Comment by RadicalInquisitor** [comment]
Yep. +1 to beg you to bring this plugin back. I just don't like seeing 'em.
